mock/py/mockbuild/package_manager.py

```diff
@@ -293,7 +295,32 @@ def initialize(self):
         self.copy_certs()
         if self.buildroot.is_bootstrap:
             self.copy_distribution_gpg_keys()
-        self.initialize_config()
+        config_content = self.initialize_config()
+
+        if not self.buildroot.is_bootstrap:
+            return
+
+        config = ConfigParser()
```

Comment:
Since dnf allows duplicate `baseurl` statements, while the last seems to win.

Reply:
And, per f2f discussion, we should also probably ignore all errors which can happen
during the attempt to parse the config.  The `baseurl=file://` support is rather rarely
used thing which shouldn't even by accident break normal `--bootstrap` use-cases.